Libs/MRML/Core/vtkMRMLMessageCollection.cxx

```diff
@@ -23,6 +23,11 @@
 // MRML includes
 #include "vtkCommand.h"
 
+namespace
+{
+  const int SEPARATOR_MESSAGE_TYPE = vtkCommand::PropertyModifiedEvent;
```

Comment:
Is the use of `vtkCommand::PropertyModifiedEvent` arbitrary ?

Reply:
Yes, completely arbitrary, I just chose an event ID that has a name that makes some sense (separator is when we switch to another group, which is kind of moving to a different property) and not likely to be used otherwise.

Reply:
I'll add a comment to explain this in a follow-up commit.